site/content/docs/commands/deploy.en.md

```diff
@@ -40,13 +64,23 @@ If the deployment fails when automatic stack rollback is disabled, you may be re
 rollback of the stack via the AWS console or AWS CLI before the next deployment.
 
 ## Examples
-
 Deploys a service named "frontend" to a "test" environment.
 ```console
- $ copilot deploy --name frontend --env test
+$ copilot deploy --name frontend --env test --deploy-env=false
```

Comment:
do they _have_ to have `--deploy-env=false` if the environment already exists?

Reply:
We  could skip that and instead make deploying the env fully optional and opt-in but It'd be much less discoverable i think

Reply:
wait - so if i have _already deployed_ the `test` environment before running `copilot deploy`, it'll still prompt me to deploy it?

Reply:
yeah. maybe i need to flop that behavior? make it opt in? or do that immediately following this in a patch?

Reply:
Yeah i remember i had similar feedback maybe in the bug bash? I feel like we can probably improve in our next release as it's not a blocker imo.